utilities/set_expired_temp_password.py

```diff
@@ -0,0 +1,132 @@
+from datetime import datetime
+import pytz
+import argparse
+import random
+import string
+import boto3
+
+###############################################################################
+#
+# usage: set_expired_temp_password.py [-h] [-u] [-d DEPLOYMENT] [-b BEFORE]
+#
+# Set user-updatable password for expired temp passwords
+#
+# options:
+#   -h, --help            show this help message and exit
+#   -u, --update          Execute password update
+#   -p, --print           Print expired users
+#   -d, --deployment DEPLOYMENT
+#                         Deployment prefix (bb, test, prod, etc)
+#   -b, --before BEFORE   YYYY-MM-DD date format to filter users
+#
+###############################################################################
+
+
+utc = pytz.UTC
+
+
+class InputError(Exception):
+    pass
+
+
+# Cognito Client
+cog_client = boto3.client("cognito-idp", region_name="us-west-2")
+
+# Get cmd line args
+parser = argparse.ArgumentParser(
+    description="Set user-updatable password for expired temp passwords"
+)
+parser.add_argument(
+    "-u", "--update", action="store_true", help="Execute password update"
+)
+parser.add_argument("-p", "--print", action="store_true", help="Print expired users")
+parser.add_argument(
+    "-d", "--deployment", type=str, help="Deployment prefix (bb, test, prod, etc)"
+)
+parser.add_argument(
+    "-b", "--before", type=str, help="YYYY-MM-DD date format to filter users"
```

Comment:
nit: it would be nice if this defaulted to more than 7 days ago, since that's the default Cognito password lifetime

Reply:
If this becomes a problem down the road, we'll probably want to integrate a "reset user" option into the Portal, and/or set up an automate task to clear out users passwords after 7 days.